querqy-for-lucene/querqy-lucene/src/main/java/querqy/lucene/contrib/rewrite/wordbreak/WordBreakCompoundRewriterFactory.java

```diff
@@ -61,7 +63,8 @@ public WordBreakCompoundRewriterFactory(final String rewriterId,
                                             final List<String> reverseCompoundTriggerWords,
                                             final boolean alwaysAddReverseCompounds,
                                             final int maxDecompoundExpansions,
-                                            final boolean verifyDecompoundCollation) {
+                                            final boolean verifyDecompoundCollation,
+                                            final List<String> protectedWords) {
```

Comment:
As with the Classic*Rewriter: reverseCompoundTriggerWords and protectedWords should be sets but reverseCompoundTriggerWords has been a list of Strings for a while. I think the List<String> got introduced by the Solr (and ES) configuration but at this level Set<? extends CharSequence> would be the best semantically.

Reply:
As above, would you like me to change this to `Iterable<String>`?